main.css

```diff
@@ -0,0 +1,500 @@
+@import url('http://fonts.cdnfonts.com/css/cocogoose');
+
+html,
+body {
+  margin: 0;
+  padding: 0;
+}
+
+header {
+  width: 100%;
+}
+
+.hamb-icon {
+  width: 30px;
+  height: 30px;
+  margin-top: 12px;
+  margin-left: 20px;
+  cursor: pointer;
+}
+
+.menu-links {
+  height: 85%;
+  position: fixed;
+  top: 0;
+  left: 0;
+  right: 0;
+  bottom: 0;
+  border-radius: 0 0 10px 10px;
+  background-color: #fff;
+  padding-top: 40px;
+  display: none;
+  flex-direction: column;
+  box-shadow: #ec5242 10px 10px 90px 40px;
+  margin: 0;
+}
+
+.menu-list {
+  margin: 20px 0;
+}
+
+.hamb-close-btn {
+  width: 35px;
+  height: 35px;
+  padding-right: 20px;
+}
+
+.menu-list.close {
+  align-self: flex-end;
+  margin-top: 0;
+}
+
+.menu-link {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
+  text-transform: uppercase;
+  font-size: 26px;
+  color: #ec5242;
+  text-decoration: none;
+  cursor: pointer;
+}
+
+ul li {
+  list-style-type: none;
+}
+
+header,
+.headline {
+  background-image: url(background1.jpg);
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.headline {
+  display: flex;
+  flex-direction: column;
+}
+
+.headline.about-me-page {
+  align-items: center;
+}
+
+.main-title::before {
+  content: '"Hello! Sharing World"';
+  text-transform: none;
+  display: block;
+  font-size: 20px;
+  font-weight: normal;
+  font-family: 'lato', sans-serif;
+}
+
+.main-title.about-me-page::before {
+  text-align: center;
+}
+
+.main-title {
+  font-family: 'COCOGOOSE', sans-serif;
+  font-weight: normal;
```

Comment:
- [ ] To match the original design better, you could consider making this `bold` instead of normal.

Original design:
![image](https://user-images.githubusercontent.com/66967274/142276818-a904b39a-8525-4f8e-9494-85d28703df5a.png)

Your project:
![image](https://user-images.githubusercontent.com/66967274/142277057-304d553a-9caf-4a44-912d-edd27f05426c.png)

Your project with a bold font-weight:
![image](https://user-images.githubusercontent.com/66967274/142276903-4b1ed3c1-b08c-4868-ae46-1abd42a4ea44.png)